test/distributed/org/apache/cassandra/distributed/test/accord/AccordLoadTest.java

```diff
@@ -75,14 +102,66 @@ public void testLoad() throws Exception
                          inFlight.acquire();
                          rateLimiter.acquire();
                          long commandStart = System.nanoTime();
-                         coordinator.executeWithResult((success, fail) -> {
-                             inFlight.release();
-                             if (fail == null) histogram.add(NANOSECONDS.toMicros(System.nanoTime() - commandStart));
-//                             else exceptions.add(fail);
-                         }, "UPDATE " + qualifiedTableName + " SET v += 1 WHERE k = ? IF EXISTS;", ConsistencyLevel.SERIAL, ConsistencyLevel.QUORUM, random.nextInt(keyCount));
+                         if (random.nextFloat() < readChance)
+                         {
+                             coordinator.executeWithResult((success, fail) -> {
+                                 inFlight.release();
+                                 if (fail == null) histogram.add(NANOSECONDS.toMicros(System.nanoTime() - commandStart));
+                                 //                             else exceptions.add(fail);
+                             }, "SELECT * FROM " + qualifiedTableName + " WHERE k = ?;", ConsistencyLevel.SERIAL, random.nextInt(keyCount));
+                         }
+                         else
+                         {
+                             coordinator.executeWithResult((success, fail) -> {
+                                 inFlight.release();
+                                 if (fail == null) histogram.add(NANOSECONDS.toMicros(System.nanoTime() - commandStart));
+    //                             else exceptions.add(fail);
+                             }, "UPDATE " + qualifiedTableName + " SET v += 1 WHERE k = ? IF EXISTS;", ConsistencyLevel.SERIAL, ConsistencyLevel.QUORUM, random.nextInt(keyCount));
+                         }
+                     }
+
+                     if ((nextRepairAt -= batchSize) <= 0)
+                     {
+                         nextRepairAt += repairInterval;
+                         System.out.println("repairing...");
+                         cluster.coordinator(1).instance().nodetool("repair", qualifiedTableName);
```

Comment:
nit:

```suggestion
                         cluster.coordinator(1).instance().nodetoolResult("repair", qualifiedTableName).asserts().success();
```

if repairs fails you won't know

Reply:
yeah, this is just for chaos